.github/workflows/version.yml

```
@@ -0,0 +1,115 @@
+jobs:
+  check-release:
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    environment: ${{ github.base_ref }}
+    permissions: read-all
+    steps:
+      - name: Get release tag
+        run: echo 'tag=${{ env.RELEASE_TAG }}' >> $GITHUB_OUTPUT
+  check-version:
+    needs: check-release
+    if: needs.check-release.outputs.tag != ''
```

actionlint (1.7.11)

[error] property "tag" is not defined in object type {}

(expression)